packages/query-sync-storage-persister/.eslintrc

```diff
@@ -1,6 +1,6 @@
 {
   "parserOptions": {
-    "project": "./tsconfig.lint.json",
+    "project": "packages/query-sync-storage-persister/tsconfig.lint.json",
```

Comment:
those changes don't seem correct, it's also why CI pipeline fails ...

Reply:
fixed.